diff --git a/CMakeLists.txt b/CMakeLists.txt
index a30e902..8c86921 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,13 +109,11 @@ endif()
 
 # As not all CMake FindXXX modules and config packages provide target support,
 # we should explicitly check for them on.
+
+include_directories(${CMAKE_INSTALL_PREFIX}/include)
 find_package(Boost REQUIRED) # almost any Boost will do
 # Boost::boost - Target for header only dependencies (Boost include directory)
-if(NOT TARGET Boost::boost)
   list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${Boost_INCLUDE_DIRS}")
-else()
-  list(APPEND COIN_TARGET_LINK_LIBRARIES Boost::boost)
-endif()
 
 if(COIN_BUILD_MAC_X11)
   find_package(X11 REQUIRED)
@@ -152,12 +150,8 @@ else()
       set(GLU_IS_PART_OF_GL 1)
     endif()
     #include_directories("${OPENGL_INCLUDE_DIR}")
-    if (NOT TARGET OpenGL::GL)
       list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${OPENGL_INCLUDE_DIR}")
       list(APPEND COIN_TARGET_LINK_LIBRARIES "${OPENGL_gl_LIBRARY}" "${OPENGL_glu_LIBRARY}")
-    else()
-      list(APPEND COIN_TARGET_LINK_LIBRARIES OpenGL::GL OpenGL::GLU)
-    endif()
   endif()
 endif()
 
@@ -173,12 +167,8 @@ if(Threads_FOUND)
     set(COIN_THREADID_TYPE pthread_t)
   endif()
   #include_directories("${THREADS_INCLUDE_DIRS}")
-  if (NOT TARGET Threads::Threads)
     list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${THREADS_INCLUDE_DIRS}")
     list(APPEND COIN_TARGET_LINK_LIBRARIES "${CMAKE_THREAD_LIBS_INIT}")
-  else()
-    list(APPEND COIN_TARGET_LINK_LIBRARIES Threads::Threads)
-  endif()
 endif()
 
 if(USE_EXTERNAL_EXPAT)
@@ -186,12 +176,8 @@ if(USE_EXTERNAL_EXPAT)
   if(EXPAT_FOUND)
     set(HAVE_EXPAT 1)
     #include_directories("${EXPAT_INCLUDE_DIRS}")
-    if (NOT TARGET EXPAT::EXPAT)
       list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${EXPAT_INCLUDE_DIRS}")
       list(APPEND COIN_TARGET_LINK_LIBRARIES "${EXPAT_LIBRARIES}")
-    else()
-      list(APPEND COIN_TARGET_LINK_LIBRARIES EXPAT::EXPAT)
-    endif()
   endif()
 else()
   if(NOT WIN32)
@@ -248,12 +234,8 @@ if(NOT ZLIB_RUNTIME_LINKING)
   if(ZLIB_FOUND)
     set(HAVE_ZLIB 1)
     #include_directories("${ZLIB_INCLUDE_DIRS}")
-    if (NOT TARGET ZLIB::ZLIB)
       list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${ZLIB_INCLUDE_DIRS}")
       list(APPEND COIN_TARGET_LINK_LIBRARIES "${ZLIB_LIBRARIES}")
-    else()
-      list(APPEND COIN_TARGET_LINK_LIBRARIES ZLIB::ZLIB)
-    endif()
   endif()
 endif()
 
@@ -262,12 +244,8 @@ if(NOT LIBBZIP2_RUNTIME_LINKING)
   if(BZip2_FOUND)
     set(HAVE_BZIP2 1)
     #include_directories("${BZIP2_INCLUDE_DIRS}")
-    if (NOT TARGET BZip2::BZip2)
       list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${BZIP2_INCLUDE_DIRS}")
       list(APPEND COIN_TARGET_LINK_LIBRARIES "${BZIP2_LIBRARIES}")
-    else()
-      list(APPEND COIN_TARGET_LINK_LIBRARIES BZip2::BZip2)
-    endif()
   endif()
 endif()
 
@@ -276,12 +254,8 @@ if(NOT FREETYPE_RUNTIME_LINKING)
   if(Freetype_FOUND)
     set(HAVE_FREETYPE 1)
     #include_directories("${FREETYPE_INCLUDE_DIRS}")
-    if (NOT TARGET Freetype::Freetype)
       list(APPEND COIN_TARGET_INCLUDE_DIRECTORIES "${FREETYPE_INCLUDE_DIRS}")
       list(APPEND COIN_TARGET_LINK_LIBRARIES "${FREETYPE_LIBRARIES}")
-    else()
-      list(APPEND COIN_TARGET_LINK_LIBRARIES Freetype::Freetype)
-    endif()
     set(HAVE_FREETYPE_H 1)
   endif()
 endif()
@@ -529,19 +503,15 @@ elseif(APPLE)
   endif()
 endif()
 
-if(NOT (HAVE_WGL OR HAVE_AGL OR HAVE_CGL OR HAVE_GLX))
-  if (NOT TARGET OpenGL::GLX)
-    list(APPEND COIN_TARGET_LINK_LIBRARIES "${OPENGL_glx_LIBRARY}")
-  else()
-    list(APPEND COIN_TARGET_LINK_LIBRARIES OpenGL::GLX)
-  endif()
-  set(CMAKE_REQUIRED_LIBRARIES ${COIN_TARGET_LINK_LIBRARIES})
-  check_cxx_source_compiles("
-    #include <GL/gl.h>
-    #include <GL/glx.h>
-    int main() { (void)glXChooseVisual(0L, 0, 0L); glEnd(); return 0; }
-  " HAVE_GLX)
-endif()
+# if(NOT (HAVE_WGL OR HAVE_AGL OR HAVE_CGL OR HAVE_GLX))
+#     list(APPEND COIN_TARGET_LINK_LIBRARIES "${OPENGL_glx_LIBRARY}")
+#   set(CMAKE_REQUIRED_LIBRARIES ${COIN_TARGET_LINK_LIBRARIES})
+#   check_cxx_source_compiles("
+#     #include <GL/gl.h>
+#     #include <GL/glx.h>
+#     int main() { (void)glXChooseVisual(0L, 0, 0L); glEnd(); return 0; }
+#   " HAVE_GLX)
+# endif()
 
 # Checks specific OpenGL configurations
 if(HAVE_WINDOWS_H)
@@ -765,4 +735,4 @@ endif()
 
 # ############################################################################
 # New CPACK section, please see the README file inside cpack.d directory.
-add_subdirectory(cpack.d)
+# add_subdirectory(cpack.d)
